MH4U Dex/HomeViewController.m

```diff
@@ -28,6 +30,21 @@ - (IBAction)resetButtonPressed:(id)sender
     [CoreDataController setShouldTriggerReloadUponRestart:YES];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [MBProgressHUD showHUDAddedTo:self.view animated:YES];
+    dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
+    dispatch_async(queue, ^{
+        [[CoreDataController sharedCDController] tryLoadSequence];
```

Comment:
It's not safe to use an `NSManagedObjectContext` instance across multiple threads and probably not safe to use it on a thread other than the one on which it was created.